test/tests-scheduling.cpp

```diff
@@ -858,3 +860,59 @@ TEST(scheduling_eval_test, spmv_fuse) {
   expected.compute();
   ASSERT_TENSOR_EQ(expected, y);
 }
+
+TEST(scheduling, divide) {
+  auto dim = 256;
+  float sparsity = 0.1;
+  Tensor<int> A("A", {dim, dim}, {Dense, Sparse});
+  Tensor<int> x("x", {dim}, {Dense});
+  IndexVar i("i"), i1("i1"), i2("i2"), j("j"), f("f"), fpos("fpos"), f0("f0"), f1("f1");
+
+  srand(59393);
+  for (int i = 0; i < dim; i++) {
+    for (int j = 0; j < dim; j++) {
+      auto rand_float = (float)rand()/(float)(RAND_MAX);
+      if (rand_float < sparsity) {
+        A.insert({i, j},((int)(rand_float * 10 / sparsity)));
+      }
+    }
+  }
+
+  for (int j = 0; j < dim; j++) {
+    float rand_float = (float)rand()/(float)(RAND_MAX);
+    x.insert({j}, ((int)(rand_float*10)));
+  }
+
+  x.pack(); A.pack();
+
+  auto test = [&](std::function<IndexStmt(IndexStmt)> f) {
```

Comment:
These test cases should be decomposed using `TEST_P`.

Reply:
I think that it is simpler for these parametrizations (that need to apply different scheduling statements which depend on the context) to be written this way than with `TEST_P`